Makefile

```
@@ -19,7 +19,7 @@
 all: build
 
 # VERSION is the version of the binary.
-VERSION:=$(shell git describe --tags --dirty)
+VERSION:=$(shell if [ -d .git ]; then echo `git describe --tags --dirty`; else echo "UNKNOWN"; fi)
```

I think we can make this optional. Silently return `UNKNOWN` doesn't seem good as well.

Making it optional, so that people build it can specify a string themselves.

---

Good suggestion. Will do.

---

Done